nav2_motion_primitives/src/back_up.cpp

```diff
@@ -45,19 +43,20 @@ nav2_tasks::TaskStatus BackUp::onRun(const nav2_tasks::BackUpCommand::SharedPtr
     RCLCPP_INFO(node_->get_logger(), "Backing up in Y and Z not supported, "
       "will only move in X.");
   }
+  command_x_ = command->x;
+  if (!robot_->getOdometry(initial_pose_)) {
```

Comment:
A suggestion for the `robot` class, maybe we should have an `isOdometryAvailable` method, here we could use it like this:

```cpp
if (!robot_->isOdometryAvailable())
```

Reply:
Oh, but I also see you actually use the `initial_pose_` never mind.